frontend/src/App.tsx

```diff
@@ -47,6 +48,10 @@ const App = () => {
                               label: "Health Check",
                               Content: <HealthCheck />,
                             },
+                            {
```

Comment:
I'd prefer this tab to appear after the person tab, but before the health check tab.